### PR TITLE
Adding python source files encodings comments

### PR DESCRIPTION
**Pull request by oransimhony**
I've added `# -*- coding: utf-8 -*-` to all python files.

Python and most editors should now identify the files as utf-8 encoded
instead of ASCII characters.

Resolves: #197 

Signed-off-by: Oran Simhony <oran.simhony@gmail.com>

### CODE REVIEW

File: tern/classes/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (nishakm) [inline tern/classes/__init__.py]: The string has a trailing whitespace.

File: tern/classes/templates/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (nishakm) [inline tern/classes/templates/__init__.py]: The string has a trailing whitespace.

File: tern/command_lib/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (nishakm) [inline tern/command_lib/__init__.py]: The string has a trailing whitespace.

File: tern/utils/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (nishakm) [inline tern/utils/__init__.py]: The string has a trailing whitespace.

**Comment by nishakm** [comment]
@oransimhony Thanks for working on this! A few alterations to PR and it'll be ready to merge:

1. Some of your comments have trailing whitespaces. Could you remove those?
2. Can you combine your commits into 1 commit?
3. I've recently merged docstring changes to the test files, so they now conflict with your `# -*- coding: utf-8 -*-` comments for those files. Could you update those?

Appreciate your time!

**Comment by oransimhony** [comment]
Hi @nishakm,
I'm having some trouble with this.
* I merged all the commits and updated the docstrings, but it still looks like there are conflicts.
* I've checked the files you mentioned and tried multiple editors but I cannot see any trailing whitespace so I cannot remove them.

I would much appreciate it if you could help me solve this problem.
Thank you!

**Comment by nishakm** [comment]
> Hi @nishakm,
> I'm having some trouble with this.
> 
> * I merged all the commits and updated the docstrings, but it still looks like there are conflicts.
> * I've checked the files you mentioned and tried multiple editors but I cannot see any trailing whitespace so I cannot remove them.
> 
> I would much appreciate it if you could help me solve this problem.
> Thank you!

I can merge this change with the fixes manually if you'd like. In the future, I'd highly recommend you follow the workflow to submit PRs here: https://github.com/nishakm/puns.

**Comment by oransimhony** [comment]
Hey!
Sorry for the late reply.
I'll try to follow the workflow you supplied in the future, thank you so much.
I'll be glad if you could merge it with the fixes yourself, sorry for the inconvenience.

**Comment by nishakm** [comment]
Sure! I'd love it if you can try to submit another PR. There are several labeled technical-debt and help-wanted issues you could work on ^_^

**Comment by nishakm** [comment]
I merged this commit manually in 83049be6b0b693a90570beac03cc71564133d959

**Pull request closed without merging**